src/game.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const readline = require('readline');
+
+const { getRandom } = require('./function.random');
+const { calculateBullsAndCows } = require('./calculateBullsAndCows');
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+const game = () => {
+  terminal.question('Guess a number: ', (number) => {
+    if (number.length !== 4 || isNaN(+number) || new Set(number).size !== 4) {
```

Comment:
You should move `4` to the specific variable. Right now, it's hard to guess what this number means

Reply:
Also, let's move these checks to some variable as well